diesel/src/migrations/mod.rs

```diff
@@ -235,14 +235,12 @@ pub fn setup_database<Conn: Connection>(conn: &Conn) -> QueryResult<usize> {
 }
 
 fn create_schema_migrations_table_if_needed<Conn: Connection>(conn: &Conn) -> QueryResult<usize> {
-    conn.silence_notices(|| {
-        conn.execute(
-            "CREATE TABLE IF NOT EXISTS __diesel_schema_migrations (
-            version VARCHAR(50) PRIMARY KEY NOT NULL,
-            run_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
-        )",
-        )
-    })
+    conn.execute(
+        "CREATE TABLE IF NOT EXISTS __diesel_schema_migrations (
+        version VARCHAR(50) PRIMARY KEY NOT NULL,
+        run_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+    )",
```

Comment:
Yeah, it's produced by `rustfmt` because it's trying not to change the meaning of the string (which this PR does). I'll see if adding some backslashes makes it less weird